### PR TITLE
Narrow usages of 037DD6 to 'primary-blue'

### PR DESCRIPTION
**Pull request by brad-decker**
replaces the raw hex value, info-blue and curious-blue to primary-blue

~Depends on #9007~

### CODE REVIEW

File: ui/app/components/ui/icon/index.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
   }
 
   &--info {
-    fill: $info-blue;
```
Comment (Gudahtt) [inline ui/app/components/ui/icon/index.scss:9]: Nit: This variable is now unused, so it could be removed as well.

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: ui/app/components/ui/icon/index.scss (hunk 1 above)
Comment (brad-decker) [reply ui/app/components/ui/icon/index.scss:9]: It looks like i missed this line when trying to split my commits. it is removed in the next PR

**Comment by brad-decker** [comment]
[Update fonts](https://trello.com/c/giqTMDWN/59-update-fonts)

**Pull request merged**